psi4/driver/p4util/util.py

```diff
@@ -138,13 +138,12 @@ def set_memory(inputval):
         units = ''
     # Handle memory given as a string
     if isinstance(inputval, basestring):
```

Comment:
`try/except` `if/else` when dealing with Python 3 is not a good idea these days IMO. It adds extra code that will need to be ripped out sooner rather than later.